easyApp/Gui/Components/PreferencesDialog.qml

```diff
@@ -211,10 +211,12 @@ EaElements.Dialog {
             }
 
             EaElements.Label {
+                visible: (ExGlobals.Constants.appName == 'easyDiffraction')
```

Comment:
- It should be `'easydiffraction'` instead of `'easyDiffraction'`
- No need for brackets 
- It is better to use strict equality operator `===` instead of `==`